src/app/CommandHandlerImpl.cpp

```diff
@@ -542,30 +505,21 @@ Status CommandHandlerImpl::ProcessGroupCommandDataIB(CommandDataIB::Parser & aCo
 
         const ConcreteCommandPath concretePath(mapping.endpoint_id, clusterId, commandId);
 
-        if (mpCallback->CommandExists(concretePath) != Status::Success)
         {
-            ChipLogDetail(DataManagement, "No command " ChipLogFormatMEI " in Cluster " ChipLogFormatMEI " on Endpoint 0x%x",
-                          ChipLogValueMEI(commandId), ChipLogValueMEI(clusterId), mapping.endpoint_id);
+            DataModel::InvokeRequest request;
 
-            continue;
-        }
+            request.path                 = concretePath;
+            request.subjectDescriptor    = GetSubjectDescriptor();
+            request.accessingFabricIndex = GetAccessingFabricIndex();
+            request.invokeFlags.Set(DataModel::InvokeFlags::kTimed, IsTimedInvoke());
 
-        {
-            Access::SubjectDescriptor subjectDescriptor = GetSubjectDescriptor();
-            Access::RequestPath requestPath{ .cluster     = concretePath.mClusterId,
-                                             .endpoint    = concretePath.mEndpointId,
-                                             .requestType = Access::RequestType::kCommandInvokeRequest,
-                                             .entityId    = concretePath.mCommandId };
-            Access::Privilege requestPrivilege = RequiredPrivilege::ForInvokeCommand(concretePath);
-            err                                = Access::GetAccessControl().Check(subjectDescriptor, requestPath, requestPrivilege);
-            if (err != CHIP_NO_ERROR)
+            Status preCheckStatus = mpCallback->ValidateCommandCanBeDispatched(request);
```

Comment:
Group commands seem to not validate `fabric scoped` nor `timed invoke` flags in previous implementations.

I am unclear why. @bzbarsky-apple @tennessee-google is this a spec requirement? Wondering if I should add that logic (i.e. pass in that the command was invoked via group handling) to the validation or not .... To me it would make sense for validation to be uniform so I left it like that here.

Reply:
It seems at least for timed we have an extra check: https://github.com/project-chip/connectedhomeip/blob/master/src/app/CommandHandlerImpl.cpp#L423C9-L423C21

I will have to decouple that one too. It also seems somewhat coupled to depend on things ordering since code changes and refactors can change things.

Reply:
So this question is now specifically about fabric scoping.

Reply:
Group commands cannot be timed-invoked. 

For fabric-scoped, I think this is a possible miss of the implementation?

Reply:
In that case I take it that "check will not hurt" so I do not need to pass in a "invoked via group" flag around to explicitly skip some checks.